<!DOCTYPE html><html><head><meta charSet="utf-8"/><meta name="viewport" content="width=device-width,initial-scale=1"/><meta name="google-site-verification" content="c8xLy6CbRSnA2A3H8CUama6rq8hjtG1c_lBvFhO_nYs"/><title>19th International Symposium on Bioinformatics Research and Applications (ISBRA 2023) October 9 - 12, 2023
Łukasiewicz Research Network - PORT Polish Center for Technology Development, Wrocław, Poland</title><meta name="description" content="The International Symposium on Bioinformatics Research and Applications (ISBRA
) provides a forum for the exchange of ideas and results among researchers, developers, and practitioners working on all aspects of bioinformatics and computational biology and their applications."/><meta property="og:title" content="ISBRA
2023- ISBRA
2023- 19th International Symposium on Bioinformatics Research and Applications"/><meta property="og:description" content="The International Symposium on Bioinformatics Research and Applications (ISBRA
) provides a forum for the exchange of ideas and results among researchers, developers, and practitioners working on all aspects of bioinformatics and computational biology and their applications."/><meta name="next-head-count" content="7"/><link rel="preload" href="ad03d55cb9cb1936.css" as="style"/><link rel="stylesheet" href="ad03d55cb9cb1936.css" data-n-g=""/><link rel="preload" href="db5d3f9304ea4a49.css" as="style"/><link rel="stylesheet" href="db5d3f9304ea4a49.css" data-n-p=""/><noscript data-n-css=""></noscript><script defer="" nomodule="" src="/ISBRA
/_next/static/chunks/polyfills-5cd94c89d3acac5f.js"></script><script src="/ISBRA
/_next/static/chunks/webpack-6f45cb54dcc402af.js" defer=""></script><script src="/ISBRA
/_next/static/chunks/framework-a87821de553db91d.js" defer=""></script><script src="/ISBRA
/_next/static/chunks/main-803fd245650d2bfd.js" defer=""></script><script src="/ISBRA
/_next/static/chunks/pages/_app-36a30e3cbeb6cb10.js" defer=""></script><script src="/ISBRA
/_next/static/chunks/pages/index-998502ba26d63c84.js" defer=""></script><script src="/ISBRA
/_next/static/4fNC6aFbeblioZZDLQGY3/_buildManifest.js" defer=""></script><script src="/ISBRA
/_next/static/4fNC6aFbeblioZZDLQGY3/_ssgManifest.js" defer=""></script><script src="/ISBRA
/_next/static/4fNC6aFbeblioZZDLQGY3/_middlewareManifest.js" defer=""></script></head><body><div id="__next"><header class="Layout_header__KD6xC"><div><h2>ISBRA
2023</h2></div></header><section class="Layout_hero__dJiuI"><div><h2>19th International Symposium on Bioinformatics Research and Applications (ISBRA 2023) October 9 - 12, 2023
Łukasiewicz Research Network - PORT Polish Center for Technology Development, Wrocław, Poland<!-- --><div></div></h2></div></section><div class="Layout_root__XpAuI"><nav class="Layout_menu__N__qO"><h2>Menu</h2><ul><li><a target="_self" href="https://mangul-lab-usc.github.io/ISBRA23
/">Home</a></li><li><a target="_self" href="https://mangul-lab-usc.github.io/ISBRA23
/dates">Important Dates</a></li><li><a target="_self" href="https://mangul-lab-usc.github.io/ISBRA23
/commitees">Committees</a></li><li><a target="_self" href="https://mangul-lab-usc.github.io/ISBRA23
/previous-meetings">Previous meetings</a></li><li><a target="_self" href="https://mangul-lab-usc.github.io/ISBRA23
/submission">Submission Guidelines</a></li><li><a target="_self" href="https://mangul-lab-usc.github.io/ISBRA23
/registration">Registration and Venue</a></li><li><a target="_self" href="https://mangul-lab-usc.github.io/ISBRA23
/agenda">Program agenda</a></li><li><a target="_self" href="https://mangul-lab-usc.github.io/ISBRA23
/keynotes">Keynote Speakers</a></li><li><a target="_self" href="https://mangul-lab-usc.github.io/ISBRA23
/papers">Call for Papers/ Abstracts</a></li></ul></nav><div class="Layout_stripe__65gNv"></div><main class="Layout_main__gbdP0"><h1>Program agenda</h1>
<p><b>The list of accepted papers:</b></p>
<ol>
  <li>Michael Souza, Nilton Maia and Carlile Lavor. The Ordered Covering Problem in Distance Geometry</li>
  <li>Gabriel Siqueira, Alexsandro Oliveira Alexandrino, Andre Rodrigues Oliveira, Géraldine Jean, Guillaume Fertin and Zanoni Dias. Approximating Rearrangement Distances with Replicas and Flexible Intergenic Regions</li>
  <li>Joyanta Basak, Ahmed Soliman, Nachiket Deo, Kenneth Haase, Anup Mathur, Krista Park, Rebecca Steorts, Daniel Weinberg, Sartaj Sahni and Sanguthevar Rajasekaran. On Computing the Jaro Similarity Between Two Strings</li>
   <li>Guy Katriel, Udi Mahanaymi, Christoph Koutschan, Doron Zeilberger, Mike Steel and Sagi Snir. Using Generating Functions to Prove Additivity of Gene-Neighborhood Based Phylogenetics
</li> 
   <li>Sumaira Zaman and Mukul S. Bansal. Reducing the impact of domain rearrangement on sequence alignment and phylogeny reconstruction</li>
   <li>Huixiu Xu, Xin Tong, Haitao Jiang, Lusheng Wang, Binhai Zhu and Daming Zhu. On Sorting by Flanked Transpositions
</li> 
   <li>Enrico Rossignolo and Matteo Comin. USTAR: Improved Compression of k-mer Sets with Counters Using De Bruijn Graphs
</li>
   <li>Ming Chen, Bin Yao, Xiujuan Lei, Chunyan Ji, Zitao Hu and Yi Pan. Predicting Comprehensive Drug-Drug Interactions by Magnetic Signed Graph Neural Network</li>
   <li>Hui Feng, Guishen Wang and Chen Cao. BiRNN-DDI:A Drug-drug Interaction Event Type Prediction Model based on Bidirectional Recurrent Neural Network and Graph to Sequence Representation</li>
   <li>Md. Tofazzal Hossain, Md. Selim Reza, Yin Peng, Shengzhong Feng and Yanjie Wei. PCPI: Prediction of circRNA and protein interaction using machine learning method
</li>
   <li>Wanyi Yang, Chuanfang Wu and Jinku Bao. PDFll: Intrinsic protein disorder and function prediction from the language of life</li>
   <li>Usama Sardar, Sarwan Ali, Muhammad Sohaib Ayub, Muhammad Shoaib, Khurram Bashir, Imdadullah Khan and Murray Patterson. Sequence-Based Nanobody-Antigen Binding Prediction
</li> 
   <li>Ahtisham Fazeel, Muhammad Nabeel Asim, Johan Trygg, Andreas Dengel and Sheraz Ahmed. Deep Learning Architectures For the Prediction of YY1-Mediated Chromatin Loops</li> 
   <li>Boxin Guan, Anqi Wang, Yahan Li, Feng Li, Jin-Xing Liu and Junliang Shang. ABCAE: Artificial Bee Colony Algorithm with Adaptive Exploitation for Epistatic Interaction Detection</li> 
   <li>Tian-Jing Qiao, Feng Li, Shasha Yuan, Ling-Yun Dai and Juan Wang. scGASI: A graph autoencoder-based single-cell integration clustering method</li> 
   <li>Shuo Xu, Liping Kang, Xingyu Bi and Xiaohui Wu. Integrative analysis of gene expression and alternative polyadenylation from single-cell RNA-seq data
</li> 
   <li>Mathieu Bolteau, Jérémie Bourdon, Laurent David and Carito Guziolowski. Inferring Boolean networks from single-cell human embryo datasets
</li> 
   <li>Tai-Ge Wang, Xiang-Zhen Kong, Sheng-Jun Li and Juan Wang. CHLPCA: Correntropy-Based Hypergraph Regularized Sparse PCA for Single-cell Type Identification</li> 
   <li>Zahra Tayebi, Akshay Juyal, Alex Zelikovsky and Murray Patterson. Simulating tumor evolution from scDNA-seq as an accumulation of both SNVs and CNAs</li>
   <li>Haijing Luan, Taiyuan Hu, Jifang Hu, Ruilin Li, Detao Ji, Jiayin He, Xiaohong Duan, Chunyan Yang, Yajun Gao, Fan Chen and Beifang Niu. Multi-Class Cancer Classification of Whole Slide Images through Transformer and Multiple Instance Learning
</li> 
   <li>Sarwan Ali, Usama Sardar, Imdadullah Khan and Murray Patterson. Efficient Sequence Embedding For SARS-CoV-2 Variants Classification</li> 
   <li>Sarwan Ali, Pin-Yu Chen and Murray Patterson. Unveiling the Robustness of Machine Learning Models in Classifying COVID-19 Spike Sequences
</li> 
   <li>Yuxia Guan, Ying An, Fengyi Guo and Jianxin Wang. MPFNet: ECG Arrhythmias Classication Based on Multi-Perspective Feature Fusion
</li>
   <li>Sarwan Ali, Haris Mansoor, Prakash Chourasia and Murray Patterson. Hist2Vec: Kernel-Based Embeddings for Biological Sequence Classification
</li> 
   <li>Yan Zhang, Xin Liu, Panrui Tang and Zuping Zhang. SGMDD: Subgraph Neural Network-Based Model for Analyzing Functional Connectivity Signatures of Major Depressive Disorder</li>
   <li>Hongyang Lei, Huazhen Huang, Bokai Yang, Guosheng Cui, Ruxin Wang, Dan Wu and Ye Li. TCSA: A Text-guided Cross-view Medical Semantic Alignment Framework for Adaptive Multi-view Visual Representation Learning</li> 
   <li>Jordan Sturtz, Richard Annan, Binhai Zhu, Xiaowen Liu and Letu Qingge. A Convolutional Denoising Autoencoder for Protein Scaffold Filling</li> 
   <li>Prakash Chourasia, Taslim Murad, Sarwan Ali and Murray Patterson. Enhancing t-SNE Performance for Biological Sequencing Data through Kernel Selection</li> 
   <li>Sarwan Ali, Prakash Chourasia and Murray Patterson. PDB2Vec: Using 3D Structural Information For Improved Protein Analysis</li> 
   <li>Ying An, Anxuan Xiong and Lin Guo. DCNN: Dual-Level Collaborative Neural Network for Imbalanced Heart Anomaly Detection</li> 
   <li>Lusheng Wang and Zhaohui Zhan. Proteoform identification for top-down tandem mass spectra: efficient algorithms for global and local alignments with peak error correction</li> 
   <li>Zhidong Yang, Hongjia Li, Dawei Zang, Renmin Han and Fa Zhang. SaID: Simulation-aware Image Denoising Pre-trained Model for Cryo-EM Micrographs</li>
   <li>Jovial Niyogisubizo, Zhao Keliang, Jintao Meng, Yi Pan, Didi Rosiyadi and Yanjie Wei. Attention-Guided Residual U-Net with SE Connection and ASPP for Watershed-based Cell Segmentation in Microscopy Images</li>
   <li>Ya Lv, Jin Liu, Pei Yang and Yi Pan. Multi-modality MRI Feature Interaction for Pseudoprogression Prediction of Glioblastoma
</li> 
   <li>Shaokai Wang, Ming Zhu and Bin Ma. NeoMS: Identification of Novel MHC-I Peptides with Tandem Mass Spectrometry</li> 
   <li>Xiaodi Hou, Guoming Sang, Zhi Liu, Xiaobo Li and Yijia Zhang. Radiology Report Generation via Visual Recalibration and Context Gating-aware
</li> 
   <li>André Salgado, Francisco Fernandes and Ana Teresa Freitas. CSA-MEM: Enhancing Circular DNA Multiple Alignment through Text Indexing Algorithms
</li> 
   <li>Hafsa Farooq, Daniel Novikov, Akshay Juyal and Alex Zelikovsky. Genetic Algorithm with Evolutionary Jumps</li> 
   <li>Carissa Bleker, Stephen Grady and Michael A. Langston. A Brief Study of Gene Co-Expression Thresholding Algorithms</li> 
   <li>Hossein Saghaian, Pavel Skums, Yurij Ionov and Alex Zelikovsky. Graph-Based Motif Discovery in Mimotope Profiles of Serum Antibody Repertoire
</li> 
   <li>Huidong Ma, Cheng Zhong, Hui Sun and Haixiang Lin. ricME: long-read based mobile element variant detection using sequence realignment and identity calculation
</li> 
   <li>Casper Asbjørn Eriksen, Jakob Lykke Andersen, Rolf Fagerberg and Daniel Merkle. Reconciling Inconsistent Molecular Structures from Biochemical Databases</li>
   <li>Gatis Melkus, Sandra Siliņa, Andrejs Sizovs, Peteris Rucevskis, Lelde Lace, Edgars Celms and Juris Viksna. Clique-based topological characterization of chromatin interaction hubs</li> 
   <li>Jakob Lykke Andersen, Sissel Banke, Rolf Fagerberg, Christoph Flamm, Daniel Merkle and Peter F. Stadler. On the Realisability of Chemical Pathways</li> 
   <li>Rui Gao, Zixue Liu, Mei Meng and Jian He. Neurogenesis-associated Protein, a Potential Prognostic Biomarker in anti-PD-1 based kidney renal clear cell carcinoma patients’ therapeutics</li> 
   <li>Bikram Sahoo and Alex Zelikovsky. Deep Learning Reveals Biological Basis of Racial Disparities in Quadruple-Negative Breast Cancer</li> 
   <li>Bikram Sahoo and Alex Zelikovsky. Exploring Racial Disparities in Triple-Negative Breast Cancer: Insights from Feature Selection Algorithms</li> 
   <li>Yulong Li, Hongming Zhu, Xiaowen Wang and Qin Liu. HetBiSyn: Predicting Anticancer Synergistic Drug Combinations Featuring Bi-perspective Drug Embedding with Heterogeneous Data</li> 
   <li>Jingjing Zhang, Md. Tofazzal Hossain, Zhen Ju, Wenhui Xi and Yanjie Wei. Identification and functional annotation of circRNAs in neuroblastoma based on bioinformatics</li> 
   <li>Xuehua Bi, Chunyang Jiang, Cheng Yan, Kai Zhao, Linlin Zhang and Jianxin Wang. Identifying miRNA-disease Associations based on Simple Graph Convolution with DropMessage and Jumping Knowledge</li>
   <li>Sarah von Loehneysen, Thomas Spicher, Yuliia Varenyk, Hua-Ting Yao, Ronny Lorenz, Ivo Hofacker and Peter F. Stadler. Phylogenetic Information as Soft Constraints in RNA Secondary Structure Prediction</li>
   <li>Rafał Stępień, Joanna Szyda, Bartosz Czech and Magda Mielczarek. The effect of transcriptomic annotations in breast cancer DGE study</li>
  </ol>
   <p></p>
   <b>The update preliminary program is here:</b>
   <a target="_self" href="https://easychair.org/smart-program/ISBRA2023/">https://easychair.org/smart-program/ISBRA2023/</a>
  
  
  
  



</main>
</div></div><script id="__NEXT_DATA__" type="application/json">{"props":{"pageProps":{}},"page":"/","query":{},"buildId":"4fNC6aFbeblioZZDLQGY3","assetPrefix":"/ISBRA
","nextExport":true,"autoExport":true,"isFallback":false,"scriptLoader":[]}</script></body></html>
